damus/Views/ProfileZoomView.swift

```diff
@@ -54,23 +54,19 @@ struct ProfileZoomView: View {
             Color("DamusDarkGrey") // Or Color("DamusBlack")
                 .edgesIgnoringSafeArea(.all)
             
-            HStack() {
-                Button {
-                    presentationMode.wrappedValue.dismiss()
-                } label: {
-                    Image(systemName: "xmark")
-                        .foregroundColor(.white)
-                        .font(.largeTitle)
-                        .frame(width: 40, height: 40)
-                        .padding(20)
-                }
+            Button {
+                presentationMode.wrappedValue.dismiss()
+            } label: {
+                Image(systemName: "xmark")
+                    .foregroundColor(.white)
+                    .font(.subheadline)
+                    .padding(.leading, 20)
             }
-            .zIndex(1)
```

Comment:
Changes look good. But we can keep the Z index as we still want the X mark to be shown in front of the image while we zoom in to the fullest!

Reply:
Also  you can close the later pr?

Reply:
Oh yeah, good call. I'll fix that and close the other 👍